config/routes.rb

```diff
@@ -8,5 +9,7 @@
   # Defines the root path route ("/")
   root 'home#index'
 
+  get 'index' => 'home#index'
```

Comment:
Why is this here for?

Reply:
modify the route root to link to the list_event and let the home index